UW Templates/Be The First (MASTER Event).html

```diff
@@ -21,7 +21,9 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">        
-        <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" type="text/css">
+        <!--[if !mso]><!-->
+      <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" type="text/css" /> 

```

Comment:
@Gonzo427 Are we now entirely moving to system sans-serif fonts on Outlook clients?

Reply:
No, this loads on all clients except Outlook. (Notice the ! in front of mso). If an Outlook user doesn't have Open Sans, then default will be sans-serif. Without that conditional, for some reason, the default becomes Times New Roman.

Reply:
Totally, I see that.  But if the font import is now only happening on non-outlook clients, are we officially saying that everything on Outlook should just be times new roman/helvetica?  We aren't even trying to load Open Sans in Outlook?

Still approving, just want to get clarification on how hard we're pushing Open Sans generally.